pymeasure/instruments/hp/hp34401A.py

```diff
@@ -77,6 +78,8 @@ def __init__(self, adapter, name="HP 34401A", **kwargs):
             asrl={'baud_rate': 9600, 'data_bits': 7, 'parity': 2},
             **kwargs
         )
+        if type(adapter) == SerialAdapter:
+            self.init_serial()
```

Comment:
The normal procedure is, that a user uses the default VISAAdapter or they create their own adapter instance. In the latter case, the user has to provide the correct settings to the adapter instance (for example the `SerialAdapter`).
Therefore, we do not add any other adapter specific things to an instrument except for the VISAAdapter via the `super().__init__()` call.

If the read/write termination is wrong, you might add it to the super init into the `asrl` dictionary.

Reply:
Will fix it this week.